ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.17.yaml

```diff
@@ -442,6 +442,17 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-private-sshkey
+- as: ibmcloud-ipi-subnet-paging-f28
+  cron: 24 0 27 * *
+  steps:
+    cluster_profile: ibmcloud-qe
+    env:
+      APPLY_ALL_SUBNETS: "no"
+      NUMBER_SUBNETS_PAIR_PER_ZONE: "9"
+      PICKUP_SUBNETS_ORDER: 2paged
```

Comment:
The job title is using `paging`, the option here is named `2paged`, `paging` v.s `paged`, pls keep consistent.